test/CMakeLists.txt

```diff
@@ -16,7 +16,15 @@ set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
 # disable fast math for ALL TESTS
-check_cxx_compiler_flag("-fno-fast-math" _fno_fast_math_option)
+foreach(_fno_fast_math_flag -fno-fast-math /clang:-fno-fast-math)
```

Comment:
I'm curious which compiler has its no-fast-math flag as `/clang:-fno-fast-math`. Could you write a comment to say which compiler this is trying to target?

Reply:
Initially this were made by me at https://github.com/oneapi-src/oneDPL/pull/729
I think that I investigated this issue on dpcpp compiler.

Reply:
Interestingly, icx-cl and dpcpp-cl do not accept `-fno-fast-math`, but do accept `/clang:-fno-fast-math`, passing the option to the underlying clang driver.  I'm not aware of another version that they accept.  This is what I am targetting.

(They both do accept `-ffast-math` though)